pynecone/components/layout/foreach.py

```diff
@@ -12,13 +12,15 @@ class Foreach(Component):
     """A component that takes in an iterable and a render function and renders a list of components."""
 
     # The iterable to create components from.
-    iterable: Var[List]
+    iterable: Var[Union[List, Dict, Tuple, Set]]
```

Comment:
We could potentially use Var[Iterable] to make it more succinct: https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable